README.md

```diff
@@ -110,6 +110,69 @@ First, you'll want to have Node.js installed. You can do this by going to [nodej
 
 10. Go to your forked repository on GitHub and [create a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
 
+## Keeping Branch Up to Date and Resolving Merge Conflicts
+
+Due to open source collaboration culture, it is common to encounter merge conflicts when contributing to an open source project. Merge conflicts usually occur when there are changes on the same line(s) in the same file(s) from two different branches.
+
+All contributors contributing to this repository will go through the same process: adding themselves to the end of the contributors array in the `.all-contributorsrc` file and generating the `README.md` file.
+
+With the same files and lines to modify, there is a high potential that you will encounter merge conflicts that you need to resolve before your pull request (PR) can be merged. It happens when maintainers have merged PRs before yours while you're working on your changes or waiting for your PR to be reviewed.
```

Comment:
```suggestion
 

Since the main purpose of the guestbook to add your name to  `.all-contributorsrc` and the `README.md` files, there is a high chance that you will encounter merge conflicts that you need to resolve before your pull request (PR) can be merged. It happens when maintainers have merged PRs before yours while you're working on your changes or waiting for your PR to be reviewed.
```